### DISCUSSION TITLE
Break HTML into multiple lines

### DISCUSSION THREAD

**Discussion by Omer-Reebelo** [vercel/next.js#55038] [Help]
### Summary

The HTML generated from nextjs/webpack shows as a single line when fetched by the client. This is the HTML code that I am referring to and not the rendering. 

My webpack and nextjs config is pretty standard. I tried using HtmlWebpackPlugin with the following but that did not help . What am I missing. 

`config.plugins.push(
          new HtmlWebpackPlugin({
            minify: {
              collapseWhitespace: false,
              removeComments: false,
              removeRedundantAttributes: false,
              removeScriptTypeAttributes: false,
              removeStyleLinkTypeAttributes: false,
              useShortDoctype: false
            }
          })
        );`

**Comment by icyJoseph** [comment]
I think the issue here is that the React methods that output the HTML strings, do so in one line, without formatting. 

I seem to remember though that someone had managed to do something about it, but I can't find it. For example you could run Prettier on the .next folder .html files.
